packages/aurora-feed/index.js

```diff
@@ -119,11 +136,14 @@ class Feed extends React.Component {
   render() {
     // Create a note for each uuid
     const uuids = Object.keys(this.state.shownNotes);
+    console.log(uuids);
```

Comment:
NIT: Avoid leaving these in if you can. 
(The linter will notice and possibly remove these for you)

Reply:
Yeah, I forgot to remove that

Reply:
No worries, happens to me all the time 😅